front/src/components/UI/AuctionView/AuctionBlock.tsx

```diff
@@ -0,0 +1,20 @@
+import { AuctionViewStyle } from "./AuctionView.styled";
+import { AuctionViewText } from "./AuctionView.styled";
+import { ProductProps } from "../ProductView/ProductBlock";
+import { AuctionViewTitle } from "./AuctionView.styled";
+
+export interface AuctionProps extends ProductProps {
+  auction_site: string;
```

Comment:
auctionSite, ale to też nie do końca prawidłowo bo sam string nie definiuje strony aukcyjnej. Co najwyzej może definiować jej nazwe albo id